.github/workflows/firebaseai.yml

```diff
@@ -36,6 +36,7 @@ jobs:
       setup_command: scripts/update_vertexai_responses.sh
 
   testapp-integration:
+    if: (github.repository == 'Firebase/firebase-ios-sdk' && github.event_name == 'schedule') || github.event_name == 'pull_request'
```

Comment:
Thanks. I don't understand why this works for forks. It seems like it only prevents the action from running nightlies on FirebasePrivate or someone else who clones. 

But I checked some history and don't see anything different.